src/room/usePTT.ts

```diff
@@ -147,8 +188,8 @@ export const usePTT = (
     }
 
     window.addEventListener("keydown", onKeyDown);
-    window.addEventListener("keyup", onKeyUp);
-    window.addEventListener("blur", onBlur);
+    //window.addEventListener("keyup", onKeyUp);
```

Comment:
Are these meant to be left commented out? Releasing spacebar no longer works without this.

Reply:
Oops - good catch. This is how I'd been testing the admin talkover without two physical computers, but forgot to put it back afterwards. 🤦